### DISCUSSION TITLE
Message sent with alias goes to spam

### DISCUSSION THREAD

**Discussion by tdesve100** [simple-login/app#1408] [Q&A] [answered]
Good day all,

I'm new to SimpleLogin and I'm still trying to understand all concepts. I've been a protonmail user for years and just tried to do the basic test proposed in tutorial : 

- I created an alias
- then I created a contact (for a gmail address I have)
- I generated a reverse-alias for this contact
- I then sent a test email to the reverse-alias from the mailbox I used to create the first alias...
- The test message went to my gmail mailbox but ended in the spam folder...

I have tried to search this forum to find an explanation. As the message is sent by SimpleLogin, I would have expected the message would be trusted by gmail servers...

I'm sure this is a stupid mistake on my side...

Thanks in advance if anyone can guide me to fix this.

**Comment by 4b15ed24f576** [comment, marked as answer]
Hi there, SimpleLogin has implemented all best practices in terms of email delivery (SPF, DKIM, DMARC). Even if our domains are trusted by email providers most of the time, we can't guarantee that all emails will arrives in the inbox since it also rely on other factors that we can not control like the content of your email or the domain that you use.

The best you could do at this point is to remove the email from the SPAM folder and declare the email as "safe" to Gmail in order to help SimpleLogin's domain deliverability.

**Reply by tdesve100** [reply]
Thanks for the comment. As this was a test message, the (absence of) content may have triggered the antispam filter... You're right.

I will continue with "real" use cases from now on.

**Reply by is-gerardo** [reply]
Hi, I'm on the same boat, twice testing: gmail -> simplelogin (reverse alias) -> yahoo -> spam :(
  I understand that SimpleLogin follows and implements best practices and that 100% delivery guarantee is not possible. But often we have no way to contact the end recipient to ask them to check their Spam thus rendering the reverse-aliasing unreliable.

For example, I know AWS SES tracks the bounce and complaint rates, if a threshold is reached then the service is stopped. Does SimpleLogin keep track of this metric? What else can be done on both ends, us the users and SimpleLogin to improve this? Are there any metrics us (users) could share with SimpleLogin? Maybe it's not a wide spread issue and only a few experience it. Thanks

**Reply by nguyenkims** [reply]
We don't have the exact percentage but from what we see from the customer support side, SimpleLogin email quite rarely ends up in the Spam folder. We do have some automatic programs (I suspect AWS SES use the same) to detect when too many of our emails are put in the Spam and these rely on email provider reports which aren't always reliable.

**Reply by Crystal1159** [reply]
> We don't have the exact percentage but from what we see from the customer support side, SimpleLogin email quite rarely ends up in the Spam folder. We do have some automatic programs (I suspect AWS SES use the same) to detect when too many of our emails are put in the Spam and these rely on email provider reports which aren't always reliable.

I don't agree.
Most of my emails end up in spam folder, and I have no response from the receiver, since noone look at their spam folder. This is why simplelogin is useless.

**Comment by Bluebubbery** [comment]
Still occurring as of August 2026, and I can rule out every content/authentication explanation given in this thread. I've filed a detailed report with redacted full headers and blocklist evidence [here](https://github.com/simple-login/app/issues/2786). SPF/DKIM/DMARC all pass with strict alignment on a p=reject domain, the body was innocuous plain correspondence, and the recipient mailbox had never marked the domain as spam — Gmail still spam-foldered it citing similarity to past spam, which points at the shared relay's Gmail-internal reputation.
